docs/source/anesthetic.rst

```diff
@@ -108,5 +108,6 @@ anesthetic.weighted\_pandas module
 .. automodule:: anesthetic.weighted_pandas
    :members:
    :undoc-members:
+   :show-inheritance:
```

Comment:
This line needs to go, otherwise it will cause autodoc to try and find documentation in parents that do not have any docs...

Reply:
Before I was using `make html SPHINXOPTS="-W --keep-going -n"` I thought you just had a sixth sense for broken docs